packages/utils/src/validate-events/validateEvent.ts

```diff
@@ -2,12 +2,15 @@ import type { MaybeString } from 'ontime-types';
 import { EndAction, TimerType, TimeStrategy } from 'ontime-types';
 
 /**
- * Check if a given value is a valid type of string, returns null otherwise
+ * Check if a given value is a valid type of string, returns the fallback otherwise
```

Comment:
```suggestion
 * Check if a given value is a valid type linkStart, returns the fallback otherwise
 * linkStart can be a string (id of an event to link) or null (unlinked)
```